tests/mu_eat_empty_args.c

```diff
@@ -0,0 +1,200 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
>icen [](start = 4, length = 4)

/poetry: mu_empty_args_test.c

Reply:
renamed

---
In reply to: [270706181](https://github.com/Azure/azure-macro-utils-c/pull/11#discussion_r270706181) [](ancestors = 270706181)